### DISCUSSION TITLE
Second job not starting

### DISCUSSION THREAD

**Discussion by dfukuba-zz** [swar/Swar-Chia-Plot-Manager#881] [Q&A]
Running into some issues with a second job starting. The first one is running fine, it's just not kicking off the second one. 

Here is the manager view: 
![image](https://user-images.githubusercontent.com/10469853/119531424-bfe1da00-bd38-11eb-9032-7d4a2c2f92ff.png)

my two jobs in the yaml config: 
![image](https://user-images.githubusercontent.com/10469853/119531494-d5570400-bd38-11eb-99a2-dff401d5d937.png)

Global settings:
![image](https://user-images.githubusercontent.com/10469853/119531546-e1db5c80-bd38-11eb-8fec-8705bda8556a.png)

Appreciate any help. Other than a few bugs here and there I'm really enjoying swar! Should have started using this sooner :)

**Comment by dfukuba-zz** [comment]
SOLVED: Simple issue. Second job had` D:` as the directory, not `D:\`

**Reply by mbabak74** [reply]
you are using you OS disk as temp disk? Better use a second ssd for that. 
you OS disk might wear out because of too many writes. Another thing is that plotting will keep the disk very busy, so there won't be much bandwith left for your OS to read and write to.
